### PR TITLE
[Infra] Add pypi and license badges

### PR DESCRIPTION
**Pull request by dominikandreasseitz**

### CODE REVIEW

**Review by madagra — APPROVED** [review]
Pre-approving with a small request.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ programs**_ with tunable qubit interaction defined on _**arbitrary register topo
 [![pre-commit](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml)
 [![tests](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml)
 [![Build documentation](https://github.com/pasqal-io/qadence/actions/workflows/build_docs.yml/badge.svg)](https://pasqal-io.github.io/qadence/latest)
```
[the remark this answers is not in the document]
Comment (Roland-djee) [reply README.md:8]: I guess it picks up the name in the action yaml file.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ programs**_ with tunable qubit interaction defined on _**arbitrary register topo
...
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Pypi]](https://badge.fury.io/py/qadence.svg)(https://pypi.org/project/qadence/)
 
 Documentation can be found [here](https://pasqal-io.github.io/qadence/latest).
```
Comment (Roland-djee) [inline README.md:12]: Please remove. The URL is already in the badge.

Comment (dominikandreasseitz) [reply README.md:12]: ?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dominikandreasseitz) [reply README.md:8]: changed

File: README.md (hunk 2 above)
Comment (Roland-djee) [reply README.md:12]: > ?

The URL to the docs is at least in three places, on the repo front page, in line 12 and in line 8. I think line 8 is sufficient.

Comment (dominikandreasseitz) [reply README.md]: removed

**Pull request merged**